packages/driver-test-support/lib/unit-suite.js

```diff
@@ -580,24 +580,11 @@ export function driverUnitTestSuite(
       d = new DriverClass();
     });
 
-    it('should say a feature is enabled when it is explicitly allowed', function () {
```

Comment:
I think we should have a test to validate that arguments without a prefix are still parsed correctly. The test can then be removed for Appium 3.

Reply:
I believe this is already covered by existing functional tests. See, for example https://github.com/appium/appium/blob/master/packages/execute-driver-plugin/test/e2e/plugin.e2e.spec.js